src/graphqlqueries/Layers.ts

```diff
@@ -0,0 +1,9 @@
+export const allLayersByIds = `query($ids: [Int]) {
+  allLayersByIds(ids: $ids) {
```

Comment:
wouldn't `layersByIds` or `layersByIdList` make more sense? since this only returns layers in `$ids`

Reply:
:+1: